content/millennium/r4/base/entities/location.md

```diff
@@ -9,9 +9,9 @@ title: Location | R4 API
 
 ## Overview
 
-The Location resource describes physical places where healthcare services are provided. In Millennium, facilities are the top level of the patient location hierarchy. A facility is also an Organization resource at which patient locations are associated. All facilities are organizations, but not all organizations are facilities. A location can also be an ambulatory patient care area like a clinic or an emergency room. The location hierarchy from highest to lowest is facility, building, nursing unit, room, and bed.
+The Location resource describes physical places where healthcare services are provided. In Cerner Millennium, facilities are the top level of the patient location hierarchy. A facility is also an Organization resource that is associated with patient locations. All facilities are organizations, but not all organizations are facilities. A location can also be an ambulatory patient care area such as a clinic or an emergency room. The location hierarchy from highest to lowest is facility, building, nursing unit, room, and bed.
```

Comment:
isn't it no longer "Cerner"? "Oracle Health Millennium Platform" or, colloquially, "Millennium"?

Reply:
We do not have the go-ahead from Legal yet to change Cerner to Oracle Health in our documentation. For now we are still following the standard of always using the full name of Cerner Millennium.